public/assets/css/style.css

```diff
@@ -710,6 +711,9 @@ button {
 }
 
 @media screen and (min-width: 680px) {
+    .svg {
+        transform: translateX(300px);
```

Comment:
what is the rational behind the value ? 
ps : the rational cannot be "I saw it on my screen" ;)